corehq/apps/custom_data_fields/edit_model.py

```diff
@@ -349,7 +349,7 @@ def save_profiles(self):
             )
             if not created and obj.has_users_assigned:
                 refresh_es_for_profile_users.delay(self.domain, obj.id)
-                bulk_sync_usercases_if_applicable(obj.definition.domain, obj.user_ids_assigned())
+                bulk_sync_usercases_if_applicable(obj.definition.domain, list(obj.user_ids_assigned()))
```

Comment:
I just did this because the old version of `user_ids_assigned` returned a list, whereas now it returns a queryset object.  Figured I'd evaluate it just in case, but looking at how it's used in `bulk_sync_usercases_if_applicable` that's probably not necessary.